ios/PolyPodApp/PolyPod/RuntimeInfo.swift

```diff
@@ -11,4 +11,45 @@ struct RuntimeInfo {
         let buildNumber = info["CFBundleVersion"] ?? "Unknown"
         return "\(marketingVersion) (\(buildNumber))"
     }
+    
+    static var isProduction: Bool {
+        if isDebug || isRunningInTestFlightEnvironment {
+            return false
+        }
+        
+        return true
+    }
+    
+    static var isDebug: Bool {
+        #if DEBUG
+            return true
+        #else
+            return false
+        #endif
+    }
+    
+    private static var isRunningInTestFlightEnvironment: Bool {
+        if isAppStoreReceiptSandbox && !hasEmbeddedMobileProvision {
+            return true
+        } else {
+            return false
+        }
+    }
+    
+    private static var isAppStoreReceiptSandbox: Bool {
+        guard let url = Bundle.main.appStoreReceiptURL else {
+            return false
+        }
+        guard url.lastPathComponent == "sandboxReceipt" else {
+            return false
+        }
+        return true
+    }
+    
+    private static var hasEmbeddedMobileProvision: Bool {
+        guard Bundle.main.path(forResource: "embedded", ofType: "mobileprovision") == nil else {
```

Comment:
Couldn't this function just be `return Bundle.main.path(forResource: "embedded", ofType: "mobileprovision") != nil`?